src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -218,7 +220,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="Common\Interop\Unix\" />
+    <Folder Condition="'$(TargetsUnix)' == 'true'" Include="Common\Interop\Unix\" />
```

Comment:
Does a Conditional Folder even mean anything?  The project view shows everything, regardless of condition.